docs/features/experiment.md

```diff
@@ -194,3 +194,54 @@ necessary metrics queries, using the `{{templates.baseline.podTemplateHash}}` an
     Rollout. This is despite the fact that the PodSpec are the same. This is intentional behavior,
     in order to allow the metrics of the Experiment's pods to be delineated and queried separately
     from the metrics of the Rollout pods.
+
+
+
+## Weighted Experiment Step with Traffic Routing
```

Comment:
instead of having as a part of same section, split into a separate section with name weights Experiemnt
and mention sub sections
ALB and SMI with runnable examples

Reply:
The experiment step feature itself doesn't change depending on which Traffic Router is used. We can add examples in the getting-started guide or the examples folders.